src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="container">
+      <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element name. Here, 'stopwatch--speed-up' seems like a modifier, but there's no indication that 'stopwatch' is present as a block on its own. Ensure that 'stopwatch' is defined as a block before using modifiers.